life/updated/02-life.adoc

```diff
@@ -1,189 +1,192 @@
+take your pleasure and be at rest.`"
+A liar he is, and ever was from the beginning. My dear children, believe him not,
+if it be your lots to be under temptations, or exercise of any kind,
+or whatever way the Lord may be pleased to lead you in
 for the trial of your faith and patience.
 The enemy will betray as many as he can, therefore look unto the Lord,
-and keep him in your remembrance, and pray unto him in your minds,
-although you cannot utter one word; know assuredly,
-that he is near to help his afflicted children at all times.
-Oh! that I had known this in the days of my ignorance, in my young and tender years,
-when the Lord was at work in my heart, and I knew it not.
-For lack of understanding, the enemy betrayed me and led me aside in those things,
-by hearkening to him and the young people who were my neighbors,
-in persuading me that it would be of great benefit to me, for I was young,
-and knew not what I might come to.
-I was left of my tender father, with hardly any friend;
+and keep Him in your remembrance, and pray unto Him in your minds.
+Although you cannot utter one word, know assuredly
+that He is near to help His afflicted children at all times.
+Oh, that I had known this in the days of my ignorance, in my young and tender years,
+when the Lord was at work in my heart, and I knew it not!
+For lack of understanding, the enemy betrayed me and led me aside.
+By hearkening to him and the young people who were my neighbors,
+he persuaded me that the world had much that would be of great benefit to me,
+for I was young, and knew not what I might come to.
+I was left, by the death of my tender father, with hardly any friends,
 and in my distress and afflictions, willing to have a little rest and comfort,
 I lent an ear to the enemy of my soul, and let my mind go forth after fine clothes.
-When it was drawn out, it went without limit;
+When it was drawn out, it went without limit,
 and when I decked myself as finely and as choice as I could,
-it would hardly give me content; for when I had one new thing, and saw another,
+it would hardly give me content. When I had one new thing, and saw a second,
 or a third, I was as desirous of it as for the former, and so ever unsatisfied.
-The lying enemy had promised me rest and peace, but could not give it;
-a liar he is and ever was, my soul is at enmity with him;
-the Lord preserve me out of his snares, and my house also forever.
+The lying enemy had promised me rest and peace, but could not give it.
+A liar he is and ever was, and my soul is at enmity with him.
+May the Lord preserve me and my house out out of his snares forever.
 
-But though he had thus drawn out my mind, the Lord did not leave me;
-for many times I had a concern what would become of me;
-and if at any time I was drawn into mirth or laughter,
+But though he had thus drawn out my mind, the Lord did not leave me.
+For many times I had a concern what would become of me.
```

Comment:
sometimes its ok to start a sentence with "For"... But here I think it sounds like she starts a new-ish subject, and it would be better to kill the "For" and just start the sentence with: Many times I....